providers/apis/polymarket/README.md

```diff
@@ -0,0 +1,94 @@
+# PolyMarket Provider
+
+Docs: https://docs.polymarket.com/
+
+Polymarket is a web3 based prediction market. This provider uses the Polymarket CLOB REST API to access the off-chain order book. 
+
+## How it Works
+
+Polymarket uses [conditional outcome tokens](https://docs.gnosis.io/conditionaltokens/), a token that represents an outcome of a specific event. All tokens in Polymarket are denominated in terms of USD.
+
+Tickers take the form of the `<market_slug>?<outcome>/USD`:
+
+Example: `WILL-BERNIE-SANDERS-WIN-THE-2024-US-PRESIDENTIAL-ELECTION?YES/USD`
```

Comment:
Is there any issue with changing the dashes to underscores? Mainly asking as dashes are used in dYdX software to separate base/quote in our ticker names.

Reply:
sure. the name isn't used anywhere in our codepaths, more of a human readable identifier for the token_id. 

updated here [switch - to _](https://github.com/skip-mev/slinky/pull/629/commits/365999df1e9c17d59cf687255a60263d29e46238)